### PR TITLE
Improve feedback. Fix important typo in jem_survey_static.yaml!

### PR DESCRIPTION
**Pull request by trey0**
This is mostly focused on improving the progress feedback.

Ideally, the progress info would come from a "smart introspective action" that can provide a better estimate of its own remaining time than the original estimate in the planning model. But that would be hard, so we don't do that.

Instead, we target a simpler semantics for the progress indicator. Now it just reports the proportion of the original estimated duration that has elapsed. So for example, if the original estimated duration was 10 minutes and we are 15 minutes into execution, it reports 150%. This is kind of dumb, but potentially handy for operators to understand at a glance that the action is running long. The fact that the progress bar is changing also tells you that the action node is ticking.

In the process of figuring this out, I found an important typo in `jem_survey_static.yaml`! So, folding that in.

### CODE REVIEW

File: astrobee/survey_manager/survey_planner/src/isaac_action_node.cpp
Comment (bcoltin) [inline astrobee/survey_manager/survey_planner/src/isaac_action_node.cpp]: Would it be possible to do this in the constructor, since (if I understand correctly?) this will get called again for each time we do the action vs. only once in the constructor?

**Review by bcoltin — APPROVED** [review]
This looks good to me but let's wait to see what Marina thinks before merging. We had been talking that the prints were too long to read things well, and maybe switching to just action and the params or a subset rather than the full command (but probably don't need to block this for that)

File: astrobee/survey_manager/survey_planner/src/isaac_action_node.cpp
Comment (trey0) [reply astrobee/survey_manager/survey_planner/src/isaac_action_node.cpp]: Good thought. (It takes < 1 second, so not very critical.)

**Comment by trey0** [comment]
> We had been talking that the prints were too long to read things well, and maybe switching to just action and the params or a subset rather than the full command (but probably don't need to block this for that)

Hm. You really want more than the action name when the two robots are executing the same action in parallel and you can't figure out which robot the feedback is about. It's possible that simply truncating the full command string at some fixed length is a reasonable compromise. Likely Marina's terminal window is a different size from mine so I can't optimize well for her.

File: astrobee/survey_manager/survey_planner/src/isaac_action_node.cpp
Comment (trey0) [reply astrobee/survey_manager/survey_planner/src/isaac_action_node.cpp]: Fixed in https://github.com/marinagmoreira/isaac/pull/2/commits/f49dd6cdc257bf30160cfeecccc0fef6b50c8900

**Comment by trey0** [comment]
PR reopened here https://github.com/nasa/isaac/pull/141

**Pull request closed without merging**
